app/services/merchant_service.rb

```diff
@@ -0,0 +1,18 @@
+class MerchantService
+
+  def self.get_all_merchants
+    response = conn.get("/merchants")
+    require "pry"; binding.pry
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.get_merchant(merchant_id)
+    response = conn.get("/merchants/#{merchant_id}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.conn
+     url = 'http://localhost:3000/api/v1'
```

Comment:
I think if you move the `/api/v1` up to the `conn.get` methods on lines 4 and 10 this will work for you and your test will pass. So here on line 15 you would have: `url = 'http://localhost:3000` and then line 10 for example would be: `'response = conn.get("/api/v1/merchants/#{merchant_id}")`